### PR TITLE
Automate building Faas Server container

### PR DESCRIPTION
**Pull request by johnmuth81**

### CODE REVIEW

**Review by sv641 — APPROVED** [review]

File: kontain-faas/server/Makefile [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+# Build the container image unpack tool for the faas server to use
+oci-image-tool:
+	# Verify that GOPATH env var is set
+	@if [ "$(GOPATH)" == "" ]; \
+	then \
+		echo "GOPATH must be in the environment"; \
+		exit 1; \
+	fi
+	rm -f go.mod go.sum
+	go get -d github.com/opencontainers/image-tools/cmd/oci-image-tool
```
Comment (sv641) [inline kontain-faas/server/Makefile:47]: Are we planning on using rawhide?
This makes me feel uncomfortable.

Comment (johnmuth81) [reply kontain-faas/server/Makefile:47]: What's rawhide?

Comment (sv641) [reply kontain-faas/server/Makefile:47]: pickup what is top of tree.

File: kontain-faas/server/Makefile [hunk 2]
```diff
@@ -0,0 +1,67 @@
+# Copyright © 2022 Kontain Inc. All rights reserved.
```
Comment (paulpopelka) [inline kontain-faas/server/Makefile:1]: 2022?  I've lost an entire year?

**Pull request merged**